src/styles/index.scss

```diff
@@ -1,3 +1,102 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+a,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 36px;
+    padding-left: 3px;
+    padding-right: 3px;
+  }
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    margin-bottom: 5px;
+    color: #060b35;
+    line-height: 18px;
+  }
+  &__code {
+    font-size: 10px;
+    color: #616070;
+    margin-bottom: 18px;
+  }
+  &__feedback {
+    width: 166px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 22px;
+  }
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  &__star {
+    background-image: url('../../images/star-active.svg');
+    background-repeat: no-repeat;
+    background-position: 50%;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+  &__star:last-child {
+    background-image: url('../../images/star.svg');
+    margin-right: 0;
+  }
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+  }
+  &__price {
+    display: flex;
+    align-items: center;
+    width: 166px;
+    margin-bottom: 16px;
+    justify-content: space-between;
+    &-name {
+      color: #616070;
+      font-size: 12px;
+    }
+    &-amount {
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+  &__button {
+    box-sizing: border-box;
+    color: #fff;
+    background-color: #00acdc;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    font-size: 14px;
```

Comment:
Font sizes should be defined using `rem` units instead of `px` to ensure better scalability and accessibility. Consider converting '14px' to the equivalent 'rem' value.